src/runtime_src/driver/xclng/drm/xocl/subdev/qdma.c

```diff
@@ -359,6 +362,23 @@ static int stream_sysfs_create(struct stream_queue *queue)
 		goto failed;
 	}
 
+	for (i = 0; i < QDMA_QSETS_MAX; i++) {
+		temp_q = queue->qdma->queues[i];
+		if (!temp_q)
+		       continue;
+		if (temp_q->qconf.c2h && temp_q->flowid == queue->flowid) {
+			xocl_info(&pdev->dev,
+				"flowid overlapped with queue %d", i);
+			return 0;
```

Comment:
do you need to goto failed?

Reply:
This is only for statistics. I am not very confident to fail the queue creation if there is flowid, route id overlap.